samza-core/src/main/java/org/apache/samza/storage/StorageRecovery.java

```diff
@@ -63,11 +62,11 @@
 
 
 /**
- * Recovers the state storages from the changelog streams and store the storages
+ * Recovers the state storage from the changelog streams and store the storages
```

Comment:
nit: might as well fix the entire doc. "Recovers the state store from the changelog stream and stores in the directory provided...."